docs/guides/tutorials/chatgpt_bot.rst

```diff
@@ -153,8 +153,7 @@ will make it easier to import them.
 
       {
         "compilerOptions": {
-    -     "target": "es5",
-    +     "target": "es6",
+          "target": "es6",
```

Comment:
We're showing the changes the user should make, so this needs to be a diff. Please revert.